app/helpers/jobs_helper.rb

```diff
@@ -1,3 +1,28 @@
 module JobsHelper
 
+	def build_ganglia_link( host, start_seconds, report_type, node_num, size )
+    OODClusters.each do |c|
+      if host.eql?(c.id.to_s)
+        if c.custom_allow?(:ganglia)
+          @ganglia_cluster = OodCluster::Servers::Ganglia.new(c.custom_config(:ganglia))
+          @ganglia_base = @ganglia_cluster.uri.to_s
+          @ganglia_server = @ganglia_cluster.opt_query.fetch(:h, '') % {h: node_num }
+          @ganglia_uri = (@ganglia_base + '&z=' + size + '&cs=' + start_seconds.to_s + '&g=' + report_type+ "&h=" + @ganglia_server).html_safe
```

Comment:
Use string interpolation instead:

```ruby
ganglia_uri = "#{ganglia_base}&z=#{size}&cs=#{...
```

And use local variables, not instance variables.